exercisePPJ_06_III/ExercisePPJ_06_III.java

```diff
@@ -0,0 +1,28 @@
+/*
+Jaki będzie rezultat poniższego programu:
+*/
+
+public class ExercisePPJ_06_III {
+	
+	public static void main(String[] args) {
+	
+	boolean x = true, z = true;
```

Comment:
try this instead
```suggestion
	boolean x , z = true;
```
does it work?